pages/contact.js

```diff
@@ -1,25 +1,39 @@
-import Link from "next/link";
 import styles from "@/styles/Home.module.css";
-
+import {useState} from "react";
```

Comment:
Due to this deployment is failing please check this.

Reply:
But how an useState statement can lead to this? @ank1traj

Reply:
This is an ESLint issue

<img width="1438" alt="image" src="https://github.com/WebSorcery/testcase-generator/assets/32160084/8e58d133-cbd4-42e9-8c3d-ee6b7ad55e10">

Reply:
<img width="306" alt="Screenshot 2023-12-27 221134" src="https://github.com/WebSorcery/testcase-generator/assets/114601400/4e955a78-fc9c-46ed-b48f-74944f7e5805">
Using same in index.js but not giving error😔

Reply:
Change this `export default function about()` to `export default function About()`. Same for Contact

Reply:
Done @ank1traj